dids/src/test/kotlin/web5/sdk/dids/DidKeyTest.kt

```diff
@@ -72,4 +77,30 @@ class DidKeyTest {
 
     }
   }
+
+  @Nested
+  inner class ImportExportTest {
+    @Test
+    fun `importing and exporting using InMemoryKeyManager works`() {
```

Comment:
I don't quite follow what behavior you're trying to test. Can you elaborate on what `works` means in this context?

Reply:
maybe...

```suggestion
    fun `InMemoryKeyManager export then re-import doesn't throw exception`() {
```